src/Module/Transaction/presentation/ListTransactions/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FlatList, ScrollView, StyleSheet, Text, View } from 'react-native';
+import { useListTransaction } from './useListTransaction';
+import { ModalFilterTransaction } from '@ModuleTransaction/Component/ModalFilterTransaction';
+import { CardTransaction } from '@ModuleTransaction/Component/CardTransaction';
+import { SearchBar } from '@component/SearchBar';
+import { ITransaction } from '@ModuleTransaction/domain/entity/ITransaction';
+
+const ItemSeparator = () => <View style={styles.separator} />;
+
+export const ListTransactionsScreen = () => {
+    const {
+        queryListTransaction,
+        goToDetail,
+        onChangeKeyword,
+        onPressFilter,
+        showModalFilter,
+        filter,
+        showingModalFilter,
+        onSelectFilter,
+        keyword,
+    } = useListTransaction();
+
+
+    return (
+        <ScrollView nestedScrollEnabled={true}>
+            <View style={styles.containerSearchBar}>
+                <SearchBar keyword={keyword} setkeyword={onChangeKeyword} onPressFilter={showingModalFilter} selectedFilter={filter.sortBy.label}/>
+            </View>
+            <ModalFilterTransaction
+                isVisible={showModalFilter}
+                selectFilter={onSelectFilter}
+                selectedFilter={filter.sortBy}
+                onDismiss={() => onPressFilter(false)}
+            />
+            <FlatList
```

Comment:
at flatList i add props
<Flatlist
...
  numColumns={1}
  maxToRenderPerBatch={20}
  updateCellsBatchingPeriod={100}
  initialNumToRender={20}
  windowSize={10}
/>

i set maxRenderPerBatch to 20 because the item of list is small and considerably light, then i set updateCellsBatchingPeriod 100 mean the cell will update every 100ms of scrolling , i also set initial render to 20 to showing 20 item at beginning, and windowsSize i set to 10, because item on screen showing 7 then 3 item ahead is pre rendered